runtime/common/src/pallet_config.rs

```diff
@@ -1,22 +1,83 @@
 // darwinia
-use crate::gov_origin::ROOT;
-use dc_primitives::{AccountId, Balance};
+use dc_primitives::*;
 // substrate
-use frame_support::{traits::LockIdentifier, PalletId};
+use sp_core::U256;
+use sp_runtime::traits::AccountIdConversion;
 use sp_std::prelude::*;
 
+/// We assume that ~5% of the block weight is consumed by `on_initialize` handlers. This is
+/// used to limit the maximal weight of a single extrinsic.
+const AVERAGE_ON_INITIALIZE_RATIO: sp_runtime::Perbill = sp_runtime::Perbill::from_percent(5);
+/// We allow `Normal` extrinsics to fill up the block up to 75%, the rest can be used by
+/// `Operational` extrinsics.
+pub const NORMAL_DISPATCH_RATIO: sp_runtime::Perbill = sp_runtime::Perbill::from_percent(75);
+/// We allow for 0.5 of a second of compute with a 12 second average block time.
```

Comment:
The current block time is 6s, please confirm this setting or update the comment.